src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -578,12 +581,12 @@ applyUpgrades v cid height
   where
     installCoinModuleAdmin = set (evalCapabilities . capModuleAdmin) $ S.singleton (ModuleName "coin" Nothing)
 
-    applyCoinV2 = applyTxs (upgradeTransactions v cid) [FlagDisableInlineMemCheck, FlagDisablePact43, FlagDisablePact45]
+    applyCoinV2 = applyTxs (upgradeTransactions v cid) [FlagDisableInlineMemCheck, FlagDisablePact43, FlagDisablePact45, FlagDisableRuntimeReturnTypeChecking]
```

Comment:
I would guess everything in `applyUpgrades` doesn't need the flag, but PactMultiChainTest would find it if it was necessary.